doc/source/examples.rst

```diff
@@ -347,37 +399,46 @@ make sure to go through its documentation.
 
 
 Learn the weights
-=================
+-----------------
 Computing the strength of associations for the data is now easy, using for
 example ``ndl.ndl`` from the
 
 
 pyndl.ndl module
-----------------
+^^^^^^^^^^^^^^^^
```

Comment:
Rendered font looks like heading, but according to the text it should be just emphasised